### PR TITLE
[24.1] Nextflow CLI run reports feature docs

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Document NF CLI run report retrieval feature (https://github.com/seqeralabs/platform/pull/5990).

### CODE REVIEW

File: platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/features_env.yml [hunk 1]
```diff
@@ -19,3 +19,8 @@
   Description: > 
     Use AWS Simple Email Service (SES) to send Seqera emails instead of SMTP.
   Value:                'Default: `false`'
+- 
+  Environment variable:            '`TOWER_ALLOW_NEXTFLOW_LOGS`'
+  Description: > 
+    Allow log and report files from Nextflow CLI runs (`-with-tower`) to be included in Seqera reports. Runs must be launched from a workspace primary compute environment.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/features_env.yml:25]:
```suggestion
    Allow log and report files from Nextflow CLI runs (`-with-tower`) to be included in Seqera reports. Runs must be launched from a workspace's primary compute environment. 
```

**Review by justinegeffen — APPROVED** [review]
Just one potential correction otherwise LGTM, approving.

**Comment by justinegeffen** [comment]
Just one comment from my side, @llewellyn-sl, thanks!

File: platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/features_env.yml (hunk 1 above)
Comment (mbosio85) [reply platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/features_env.yml:25]: the logs will not be included in the reports. Simply, they will be made available in the UI vs not being accessible.  
The runs are launched locally on HPC or AWS without using any Tower CE
What happens is that the files (logs and reports) must be accessible to the primary CE in the workspace so tower will be able to fetch them successfully.

File: platform_versioned_docs/version-24.1.0/reports/overview.mdx [hunk 2]
```diff
@@ -30,6 +30,12 @@ Create a config file that defines the paths to a selection of output files publi
 Any configuration provided in the interface will override configuration supplied in the pipeline repository.
 :::
 
+### Configure reports for Nextflow CLI runs 
+
+Seqera can manage the reports and log files for pipeline runs launched with Nextflow CLI (`nextflow run <pipeline> -with-tower`) when the run is launched with a workspace primary compute environment. Specify the Seqera workspace from which to run the pipeline by setting the `TOWER_WORKSPACE_ID` environment variable. This workspace must have a primary compute environment set. Reports are listed under the **Reports** tab on the run details page. 
+
```
Comment (mbosio85) [inline platform_versioned_docs/version-24.1.0/reports/overview.mdx:36]: The run is not launched with a CE from platform . The run is launched with `nextflow run ... -with-tower` then the logs and reports will be visualized if the primary CE has access to those flies.

File: platform_versioned_docs/version-24.1.0/reports/overview.mdx [hunk 3]
```diff
@@ -30,6 +30,12 @@ Create a config file that defines the paths to a selection of output files publi
...
+Execution logs for pipelines run with a primary compute environment are available in the **Reports** tab by default. To specify additional report files to be made available, your pipeline repository root folder must include a `tower.yml` file that specifies the files to be included (see below).
```
Comment (mbosio85) [inline platform_versioned_docs/version-24.1.0/reports/overview.mdx:37]: This is not true either. The logs are in the logs tab and not in the reports. The reports tab will include only the reports.

**Review by mbosio85 — COMMENTED** [review]
Left a couple of comments on  things that are not 100% correct

File: platform_versioned_docs/version-24.1.0/reports/overview.mdx (hunk 2 above)
Comment (llewellyn-sl) [reply platform_versioned_docs/version-24.1.0/reports/overview.mdx:36]: Right, but it's executed with a CE that is also a primary CE in SP. I've adjusted the language a bit. Let me know if this is clearer.

**Review by mbosio85 — APPROVED** [review]
LGTM

**Comment by markpanganiban** [comment]
Hi, I've been testing it and can have stumbled upon @jordeu comment [here](https://github.com/seqeralabs/platform/pull/5990#issuecomment-1960964626),
> If you choose a CE that is an AWS Batch, then the global Nextflow logs (in a local path) do not work, but all the other things work as expected.

When using AWS Batch as CE, I can see that the reports are uploaded but the logs are not. Tower seems like it's looking for the local /home path as an S3 folder which is probably why it didn't work.
image

The question is, do we want to call this out on our documentation? Using slurm and running it within the same cluster works fine.

**Pull request merged**